server/index.js

```diff
@@ -50,7 +50,45 @@ app.patch('/api/reviews', function(req, res) {
         res.status(200).json(result);
       });
     });
+});
+
+// Created a new POST route
+
+app.post('/api/reviews', (req, res) => {
+  // console.log(req.body);
+  const review = req.body;
+  Review.create(review)
+  .then(() => {
+    res.send('successfully posted review');
+  })
+  .catch(error => console.log(error))
+});
+
+
+// Created a new PUT route
 
+app.put('/api/reviews', (req, res) => {
+  // console.log(req.query.id);
+  const id = req.query.id;
+  const body = req.body;
+  Review.update({ id }, body)
+  .then(() => {
+    res.send('successfully updated');
+  })
+  .catch(error => console.log(error))

```

Comment:
I think you might be missing a semicolon after the catch block, and I think the chains of a promise should be indented further.